### PR TITLE
Update wine{unstable,staging,gecko,mono,winetricks} 

### PR DESCRIPTION
**Pull request by lovesegfault**
###### Motivation for this change
I noticed it was out of date

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by lovesegfault** [comment]
(Still building locally)

**Comment by lovesegfault** [comment]
@GrahamcOfBorg build wine

**Comment by lovesegfault** [comment]
@GrahamcOfBorg build wineFull

File: pkgs/applications/audio/airwave/default.nix [hunk 1]
```diff
@@ -14,11 +14,10 @@ let
   };
 
   vst-sdk = stdenv.mkDerivation rec {
-    name = "vstsdk368_08_11_2017_build_121";
-    src = requireFile {
-      name = "${name}.zip";
-      url = "http://www.steinberg.net/en/company/developers.html";
-      sha256 = "e0f235d8826d70f1ae0ae5929cd198acae1ecff74612fde5c60cbfb45c2f4a70";
```
Comment (avnik) [inline pkgs/applications/audio/airwave/default.nix]: Looks like unrelated commit ;)

**Review by avnik — CHANGES_REQUESTED** [review]
Update of wine itself (outside unrelated commit) is ok
Could you remove extra commit, please?

File: pkgs/applications/audio/airwave/default.nix (hunk 1 above)
Comment (lovesegfault) [reply pkgs/applications/audio/airwave/default.nix]: Yeah, I will, I accidentally bundled it :)

Comment (lovesegfault) [reply pkgs/applications/audio/airwave/default.nix]: Done

**Review by worldofpeace — APPROVED** [review]

**Review by avnik — APPROVED** [review]
LGTM

**Review by tadeokondrak — APPROVED** [review]
